dataprofiler/profilers/profile_builder.py

```diff
@@ -277,16 +287,22 @@ def get_base_props_and_clean_null_params(self, df_series, sample_size,
         # Pandas reads empty values in the csv files as nan
         df_series = df_series.apply(str)
 
-        sample_ind_generator = utils.shuffle_in_chunks(
-            len_df, chunk_size=sample_size)
 
+        # Select generator depending if sample_ids availablity
+        if sample_ids is None:
```

Comment:
I dont think we have any tests that explore this condition. Can we make one and just make sure the profile is operating appropriately? Think the PR looks good, just needs some kind of test